examples/IRMQTTServer/IRMQTTServer.ino

```diff
@@ -3058,17 +2974,17 @@ bool sendClimate(const stdAc::state_t prev, const stdAc::state_t next,
   if (prev.fanspeed != next.fanspeed || forceMQTT) {
     diff = true;
     success &= sendString(topic_prefix + KEY_FANSPEED,
-                          fanspeedToString(next.fanspeed), retain);
+                          IRac::fanspeedToString(next.fanspeed), retain);
   }
   if (prev.swingv != next.swingv || forceMQTT) {
     diff = true;
     success &= sendString(topic_prefix + KEY_SWINGV,
-                          swingvToString(next.swingv), retain);
+                          IRac::swingvToString(next.swingv), retain);
```

Comment:
May i ask why IRac:: is needed?

Reply:
They are a [static member function](https://www.tutorialspoint.com/cplusplus/cpp_static_members.htm) of the `IRac` class now after the move. The class doesn't need to be instantiated, so it shouldn't take up extra memory/prog space. Plus it keeps the namespace a bit cleaner.

Basically, they are standalone functions that don't need the class object, but are associated with it.